globus_sdk/auth/token_response.py

```diff
@@ -56,50 +58,6 @@ def by_resource_server(self):
         """
         return self._by_resource_server
 
-    @property
-    def expires_at_seconds(self):
-        """
-        A POSIX timestamp representation of the expiration
-        time for the top-level token in the response.
-        """
-        top_level_token = self._by_resource_server[self.resource_server]
-        return top_level_token['expires_at_seconds']
-
-    @property
-    def expires_in(self):
-        """
-        The ``expires_in`` value for the top-level token in the response.
-        """
-        return int(self.expires_at_seconds - time.time())
-
-    @property
-    def access_token(self):
-        """
-        The ``access_token`` value for the top-level token in the response.
-        """
-        return self['access_token']
-
-    @property
-    def refresh_token(self):
-        """
-        The ``refresh_token`` value for the top-level token in the response.
-        """
-        return self.get('refresh_token')
-
-    @property
-    def resource_server(self):
-        """
-        The ``resource_server`` value for the top-level token in the response.
-        """
-        return self['resource_server']
-
-    @property
-    def other_tokens(self):
-        """
-        The array of tokens other than the top-level one.
-        """
-        return self['other_tokens']
-
```

Comment:
We definitely have some internal usage of these properties that will need to be refactored if they get dropped.